internal/core/operations/healthcontroller/health_controller_executor.go

```diff
@@ -208,8 +208,13 @@ func (ex *SchedulerHealthControllerExecutor) ensureDesiredAmountOfInstances(ctx
 	return nil
 }
 
-func (ex *SchedulerHealthControllerExecutor) findAvailableAndExpiredRooms(ctx context.Context, op *operation.Operation, gameRoomsIDs []string) (availableRoomsIDs, expiredRoomsIDs []string) {
+func (ex *SchedulerHealthControllerExecutor) findAvailableAndExpiredRooms(ctx context.Context, op *operation.Operation, gameRoomsIDs []string, instances []*game_room.Instance) (availableRoomsIDs, expiredRoomsIDs []string) {
 	for _, gameRoomID := range gameRoomsIDs {
+		if !ex.isGameRoomInstanceReady(gameRoomID, instances) {
```

Comment:
If an instance is in CrashLoop, its status will be `error`, and it will never be deleted by maestro since we are considering it as available